rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -185,11 +184,126 @@ local function handle_certs()
   end
 end
 
+local function get_backends_with_name(name)
+  local backends_data = configuration_data:get(name)
+
+  local backends, err = cjson.decode(backends_data)
+  if not backends_data then
+    ngx.log(ngx.ERR, "could not parse backends data: ", err)
+    return
+  end
+  return backends
+end
+
+local function get_backend_bucket_names()
+  local names = configuration_data:get("backend_bucket_names")
+  if not names then
+    return
+  end
+  local backend_bucket_names, err = cjson.decode(names)
+  if not backend_bucket_names then
+    ngx.log(ngx.ERR, "could not parse backend bucket names data: ", err)
+    return
+  end
+
+  return backend_bucket_names
+end
+
+local function get_backends(is_init_call)
+  ngx.log(ngx.DEBUG, "start get bucket name")
+  local bucket_names = get_backend_bucket_names()
+  if not bucket_names then
+    ngx.log(ngx.WARN, "bucket name not found")
+    return
+  end
+  local all_backends = {}
+  for _, name in ipairs(bucket_names) do
+    local backends = get_backends_with_name(name)
+    for _, v in ipairs(backends) do
+      table.insert(all_backends, v)
+    end
+
+    if not is_init_call then
```

Comment:
Is it necessary here?

Reply:
Yes, when nginx  reload , it is necessary to force the cpu to achieve the effect of fast data synchronization to other child processes

Reply:
the arg name is `force` is better than `is_init_call ` ? I think have beast arg name, I haven't thought of it yet.

Reply:
I think it's ok to keep the current name